pom.xml

```diff
@@ -27,18 +30,27 @@
         <org.mozilla.rhino.version>1.7.14</org.mozilla.rhino.version>
```

Comment:
Are we using rhino? As far as I know, it had some problems with Java 17.

Reply:
Rhino doesn't show up in the dependency tree because it's excluded:
```xml
<exclusion>
  <groupId>org.mozilla</groupId>  <!-- org.mule.modules:mule-json-module:2.5.0-SNAPSHOT, line 87 -->
  <artifactId>rhino</artifactId>  <!-- org.mule.modules:mule-json-module:2.5.0-SNAPSHOT, line 88 -->
</exclusion>
```

The property is therefore not needed so it can be removed.